tests/library/route-web-socket.spec.ts

```diff
@@ -33,11 +34,11 @@ function withResolvers<T = void>() {
   return { promise, resolve };
 }
 
-async function setupWS(target: Page | Frame, port: number, binaryType: 'blob' | 'arraybuffer', protocols?: string | string[]) {
-  await target.goto('about:blank');
-  await target.evaluate(({ port, binaryType, protocols }) => {
+async function setupWS(target: Page | Frame, server: TestServer, binaryType: 'blob' | 'arraybuffer', protocols?: string | string[], relativeURL?: boolean) {
+  await target.goto(server.EMPTY_PAGE);
+  await target.evaluate(({ port, binaryType, protocols, relativeURL }) => {
     window.log = [];
-    window.ws = new WebSocket('ws://localhost:' + port + '/ws', protocols);
+    window.ws = new WebSocket(relativeURL ? '/ws' : 'ws://localhost:' + port + '/ws', protocols);
```

Comment:
Nit: you can compute url outside eval and pass just the url, instead of 2 params.